lib/ecto/changeset.ex

```diff
@@ -454,6 +454,43 @@ defmodule Ecto.Changeset do
     end)
   end
 
+  @doc """
+  Returns true if a field was changed in a changeset.
+
+  ## Options
+
+    * `:to` - Check if the field was changed to a specific value
+    * `:from` - Check if the field was changed from a specific value
+
+  ## Examples
+
+      iex> post = %Post{title: "Foo", body: "Old"}
+      iex> changeset = change(post, %{title: "New title", body: "Old"})
+
+      iex> changed?(changeset, :body)
+      false
+
+      iex> changed?(changeset, :title)
+      true
+
+      iex> changed?(changeset, :title, to: "NEW TITLE")
+      false
+  """
+  @spec changed?(t, atom, Keyword.t) :: boolean
+  def changed?(%Changeset{} = changeset, field, opts \\ []) when is_atom(field) do
+    case fetch_change(changeset, field) do
+      {:ok, new_value} ->
+        Enum.all?(opts, fn
+          {:from, from} -> Map.get(changeset.data, field) == from
+          {:to, to} -> new_value == to
```

Comment:
Another note: should do semantic comparison for `:from` and `:to`

Reply:
Because of Decimal and stuff, right? Good call.

Reply:
@v0idpwn ping just in case you forgot. if you are busy, no worries, i would still wait 2-4 weeks for a new release.